### PR TITLE
api tests for bootcamp task 3

### PR DESCRIPTION
**Pull request by RafalRymek**
Bootcamp API tests covered all endpoints from exercise

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# bootcamp_3_api_tests
+# Bootcamp API automated tests
+
 Automated API tests to cover endpoints from https://reqres.in/ resource
+
```
Comment (asket888) [inline README.md:4]: Changeable variables in the documentation are marked by {}.
Please, replace 'https://reqres.in/ resource' to 'https://reqres.in/{resource}'

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,7 @@ __pycache__/
 *.so
 
 # Distribution / packaging
```
Comment (asket888) [inline .gitignore:4]: Remove all dependencies from .gitignore, and leave only specific for your project. Don't use random .gitignore from the internet

**Review by asket888 — CHANGES_REQUESTED** [review]

**Comment by asket888** [comment]
1) Following your README.md I can't run the tests
Screenshot 2021-04-08 at 14 55 25

2) Have you used pipenv for this task? You have no Pipfile and Pipfile.lock in your repo

File: .gitignore (hunk 2 above)
Comment (RafalRymek) [reply .gitignore:4]: It is gitignore which I need to choose from GitHub :) But ok I will fix it

File: tests/test_api_tests_pytest.py [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+def test_get_list_users():
+    # setup
+    get_list_users_url = "https://reqres.in/api/users?page=2"
+    # actions
+    response = requests.get(url=get_list_users_url)
+    # results
+    assert_that(response.status_code, is_(equal_to(200)), reason="Wrong status code")
+    assert_that(response.json()["data"][0]["email"], is_(equal_to("michael.lawson@reqres.in")), reason="Wrong email")
+    assert_that(response.json()["data"][0]["first_name"], has_string("Michael"), reason="Wrong name")
```
Comment (asket888) [inline tests/test_api_tests_pytest.py]: In all assertions you use the same structure `response.json()["data"]`. What do you think if we will set a special variable to get the 'data' tag only one time?

e.g:
response = requests.get(url=get_list_users_url)
data = response.json()["data"][0]
assert_that(data["email"], is_(equal_to("michael.lawson@reqres.in")), reason="Wrong email")
assert_that(data["first_name"], has_string("Michael"), reason="Wrong name")

**Review by asket888 — CHANGES_REQUESTED** [review]

**Review by asket888 — APPROVED** [review]

**Pull request merged**